### ISSUE TITLE
Midi not working after a few days.

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#888]
In GitLab by @VanB_94 on Jun 21, 2022, 24:09

Hey, I'm pretty new to Foundry and the first mod I installed was Midi because I watched some vidoes about it. I tested it, customized the options and everything worked fine. I installed some mods and after that it stilled worked fine. Since yesterday it does not work anymore. I figured maybe it's because of a mod, so I deactivated all mods except for midi and it still didn't work. I even reinstalled midi to set it back to default settings. Still doesn't work.![Unbenannt](https://my-gitlab-bucket.s3.amazonaws.com/640711674/3f9016bd78b9ccb4960bb8d25e410f050cadf9c3aa42b3852977882104f5819e/Unbenannt.PNG) This are the mods I have installed.

And this are my settings: [fvtt-midi-qol-settings.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/8d264da78e6be3c9ac00a26508cbc3fd32ef25f26063de07ae31378ccbd3e14c/fvtt-midi-qol-settings.json)

I hope you can help me my next session is on friday.

**Comment by tposney** [comment]
Can you install and activate convenient effects - the 0.9.58 release has a bug that it requires that to be installed - and let me know if that fixes the problem. Otherwise check the error console and see what errors are there.

**Comment by tposney** [comment]
In GitLab by @VanB_94 on Jun 21, 2022, 19:28

This solved the issue. Thank you!

**Comment by tposney** [comment]
In GitLab by @VanB_94 on Jun 21, 2022, 22:14

as long s as the ticket is still open:

After I got it to work again I cant figure out how to hide this chatboxes from my Players. As far as I remember this cards didnt show for my players before I had to reinstall. Feeling pretty stupid right now. This are my current settings. But maybe its another module that screws with me.

![unknown](https://my-gitlab-bucket.s3.amazonaws.com/640711674/96f634f7aaa62588d8cfbc11bd6123c58dde70258a4f38a625803649447f38b7/unknown.png) left is GM Client right side is Player client.

This are my current settings
[fvtt-midi-qol-settings.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/85e666c18a1cf8d9314ac178ae1605d6491cf6cad5a354ffbe4f957198e384da/fvtt-midi-qol-settings.json)
